packages/api-explorer/lib/create-docs.js

```diff
@@ -41,6 +41,15 @@ module.exports = (oas, apiSetting) => {
 
       const tag = operation.tags && operation.tags.length ? operation.tags[0] : path;
       if (!docs.find(category => category.slug === tag && category.type === 'basic')) {
+        // If the method present isn't an HTTP method that we're aware of, ignore it. This can
+        // happen in the case of an API definition using something like common parameters.
+        // https://swagger.io/docs/specification/describing-parameters
+        if (
+          !['get', 'post', 'put', 'patch', 'delete', 'head', 'options', 'trace'].includes(method)
```

Comment:
How does this work when common parameters are included via an OAS file from ReadMe? `create-docs` is only used within the demo app.

Reply:
That's a good point...

I'll do some further testing and send it back for review.

Reply:
I *think* the previous approach was sound, but it was just broken in that it excluded regular pages, and not common parameter endpoints?